uint256.go

```diff
@@ -142,5 +143,5 @@ func New(num ...uint64) Uint256 {
 	for i, k := skipBy(it, getStart(), LENGTH-len(num)), 0; comp(i) && k < LENGTH; i, k = it(i), k+1 {
 		v[i] = num[k]
 	}
-	return v
+	return v, nil
```

Comment:
If an exception is not expected is it always better to require the caller to handle the second return value? (Admittedly, this does seem more go-like to me, though)

Reply:
You can't return without all the return values set, none are optional.